streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -220,6 +221,50 @@ public void shouldRestoreAll() throws Exception {
             "c");
     }
 
+    @Test
+    public void shouldPutOnlyIfAbsentValue() throws Exception {
+        subject.init(context, subject);
```

Comment:
Can we change `subject` to `rockDdStore` -- it's a weird name IMHO.

Reply:
agreed, I don't see why not